tests/Testcontainers.Minio.Tests/MinioContainerTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.IO;
+using Amazon.S3;
+using Amazon.S3.Model;
+
+namespace TestContainers.Minio.Tests.Container;
+
+public sealed class MinioContainerTests : IAsyncLifetime
+{
+    private readonly MinioContainer _minioTestcontainer;
+
+    public MinioContainerTests()
+    {
+        _minioTestcontainer = new MinioBuilder().Build();
+    }
+
+    [Fact]
+    public async Task TestMinio()
+    {
+        const string bucketName = "somebucket";
+        await _minioTestcontainer.StartAsync();
+        var config = new AmazonS3Config
```

Comment:
```suggestion
        const string bucketName = "somebucket";

        var config = new AmazonS3Config
```

No reason to start the container, it is done in `InitializeAsync()`.

Reply:
fixed